rosdep/sources.list.d/20-default.list

```diff
@@ -1,10 +1,10 @@
 # os-specific listings first
-yaml https://raw.githubusercontent.com/ros/rosdistro/master/rosdep/osx-homebrew.yaml osx
+yaml https://raw.githubusercontent.com/jpardeiro/rosdistro/add_deps/rosdep/osx-homebrew.yaml osx
```

Comment:
The changes to this file likely shouldn't be included in this PR

Reply:
You are absolutely right, sorry for the mistake. I just reverted the lines and rebased the branch,